<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_action">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>160</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_start">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to &lt;span style=&quot; font-weight:600;&quot;&gt;start &lt;/span&gt;conversion of selected *.csv files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_separator">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Separator:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_separator">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Key-in separation sign e.g. ; or ,&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_separator_change">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to &lt;span style=&quot; font-weight:600;&quot;&gt;change separation sign&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_target_directory">
         <property name="text">
          <string>Target directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_target_directory">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_target_dir_change">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to &lt;span style=&quot; font-weight:600;&quot;&gt;change target directory&lt;/span&gt; for output files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_target_dir_open">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to &lt;span style=&quot; font-weight:600;&quot;&gt;open target tiregtory&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_progress">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Progress preview</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_progress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionOpen_directory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionStart_conversion"/>
    <addaction name="separator"/>
    <addaction name="actionChange_separator"/>
    <addaction name="separator"/>
    <addaction name="actionChange_target_directory"/>
    <addaction name="actionOpen_target_directory"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_directory">
   <property name="text">
    <string>Open directory</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file\-s</string>
   </property>
  </action>
  <action name="actionStart_conversion">
   <property name="text">
    <string>Start conversion</string>
   </property>
  </action>
  <action name="actionChange_separator">
   <property name="text">
    <string>Change separator</string>
   </property>
  </action>
  <action name="actionChange_target_directory">
   <property name="text">
    <string>Change target directory</string>
   </property>
  </action>
  <action name="actionOpen_target_directory">
   <property name="text">
    <string>Open target directory</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>textEdit_progress</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
